dashboards/victoriametrics-cluster.json

```diff
@@ -3581,111 +3581,6 @@
           "title": "Assisted merges ($instance)",
           "type": "timeseries"
         },
-        {
```

Comment:
It makes sense to port `Rows ignored for last 1h` panel to vmagent, since it can ignore series now.